common/config/rush/pnpm-lock.yaml

```diff
@@ -4857,12 +5255,24 @@ packages:
     engines: {node: ^10.13.0 || ^12.13.0 || ^14.15.0 || >=15.0.0}
     dependencies:
       '@jest/types': 27.5.1
-      '@types/node': 16.18.7
+      '@types/node': 18.11.13
       chalk: 4.1.2
```

Comment:
We support node 16 and 18, does this potentially break on node 16

Reply:
It's generally more the other way around (due to backwards compatibility), if we put 16 here but also support 18, it could throw errors since it's not up to date with the newest types.